lib/control/v1/signature.js

```diff
@@ -21,7 +21,7 @@ class Server {
   }
 
   start() {
-    this.process = CP.spawn(this.path, ['-p', this.port]);
+    this.process = bundledGemSpawn(this.path, ['-p', this.port]);
```

Comment:
This still isn't going to work right as far as I can see, because 1) the gems aren't being installed anywhere, and 2) bundler isn't being run in the correct working directory.

Something like

```
CP.spawn('/usr/local/bin/bundler', ['exec', 'bin/server'], {
  cwd: Path.resolve(__directory, '../../../pkcs7')
})
```
 should work.